modules/core/src/main/java/org/apache/ignite/internal/util/nio/ssl/GridNioSslFilter.java

```diff
@@ -72,6 +76,18 @@
     /** Whether direct mode is used. */
     private boolean directMode;
 
+    /** The name of the metric that provides histogram of SSL handshake duration. */
+    public static final String SSL_HANDSHAKE_DURATION_HISTOGRAM_METRIC_NAME = "sslHandshakeDurationHistogram";
```

Comment:
Most metric names start with a capital letter. Should we rename these metrics too?

Reply:
Done.